pkg/kubectl/cmd/taint.go

```diff
@@ -102,7 +91,8 @@ func NewCmdTaint(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 				cmdutil.CheckErr(err)
 			}
 		},
-		ValidArgs: validArgs,
+		ValidArgs:  []string{"node"},
+		ArgAliases: []string{"nodes"},
```

Comment:
It doesn't make sense to support the resource name but not its alias. cc: @kubernetes/kubectl